examples/ui-demo/src/app/config.tsx

```diff
@@ -85,8 +85,18 @@ export const alchemyConfig = () =>
           [{ type: "email" }],
           [
             { type: "passkey" },
-            { type: "social", authProviderId: "google", mode: "popup" },
-            { type: "social", authProviderId: "facebook", mode: "popup" },
+            {
+              type: "social",
+              authProviderId: "google",
+              mode: "redirect",
+              redirectUrl: "http://localhost:3000",
```

Comment:
to update to demo.alchemy.com before merging

Reply:
^scratch this, will switch `mode` back to popup before merging, so `redirectUrl` will not be needed